src/components/SidebarMenu.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { MutableRefObject } from "react";
+import styles from "../styles/sidebarmenu.module.css";
+
+interface SidebarProps {
+  refs: MutableRefObject<HTMLElement | undefined>[];
+}
+
+export const SidebarMenu = ({ refs }: SidebarProps) => {
+  return (
+    <div className={styles.wrapper}>
+      <h4>Your new gang</h4>
+      <ul className={styles.ul}>
+        <a
+          ref={refs[0] as MutableRefObject<HTMLAnchorElement>}
+          href="#polar bear"
+        >
+          <li className="active">Polar Bear</li>
+        </a>
+        <a ref={refs[1] as MutableRefObject<HTMLAnchorElement>} href="#cheetah">
```

Comment:
**nitpick:** Casting types like this is a bad practice, as it can change in time, and you will don't know it - such implementation can crash your app